apis/services/v1alpha1/auth_types.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/opendatahub-io/opendatahub-operator/v2/apis/common"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	AuthServiceName = "auth"
+	AuthInstanceName = "auth"
+	AuthKind         = "Auth"
+)
+
+// AuthSpec defines the desired state of Auth
+type AuthSpec struct {
+	AdminGroups []string `json:"AdminGroups"`
+	AllowedGroups []string `json:"AllowedGroups"`
```

Comment:
the name of the json field are lower camelCase

Reply:
:+1: